draft-ietf-quic-transport.md

```diff
@@ -4590,8 +4595,11 @@ preferred_address (0x000d):
 active_connection_id_limit (0x000e):
 
 : The maximum number of connection IDs from the peer that an endpoint is willing
-  to store. This value includes only connection IDs sent in NEW_CONNECTION_ID
-  frames. If this parameter is absent, a default of 0 is assumed.
+  to store. This value includes connection IDs sent in NEW_CONNECTION_ID frames,
+  the connection ID used during the handshake as well as the connection ID
+  provided in the preferred_address transport parameter.
+  If the active_connection_id_limit parameter is absent, a default of 2 is
+  assumed. Values below 2 are invalid.
```

Comment:
The default and minimal values of 2 look odd, when using a zero-length CID.

Maybe rephrase to:
> Unless a zero-length connection ID is being used, the value of the active_connection_id_limit parameter MUST be no less than 2. If the active_connection_id_limit parameter is absent, a default of 2 is
   assumed.
> 
> When a zero-length connection ID is being used, the active_connection_id_limit parameter MUST NOT be sent.

The rationale behind the last sentence is that it disputable if we consider CIDL=0 as the absence of CID or as the use of zero-length CID. The value of active_connection_id_limit becomes zero or one based on the view you choose. Hence my preference to require absence of the TP when a zero-length CID is used.

Reply:
Thanks for catching this. I updated the PR.